### PR TITLE
Package updates and new babel config and approach

### PR DESCRIPTION
**Pull request by illepic**

### CODE REVIEW

File: particle.js [hunk 1]
```diff
@@ -66,19 +66,6 @@ const cssModes = {
...
-/**
- * Polyfill ALL modern JS to support for browsers within .browserslistrc
- *
- * @param {string} entry
- * @returns {{entry: {}}}
- */
-const entryPrepend = entry => ({
-  // See webpack.[app].js for more entry points
-  entry: {
-    [entry]: ['@babel/polyfill'],
```
Comment (scottalan) [inline particle.js]: Trying to follow along...

This is due to https://babeljs.io/docs/en/babel-preset-env#usebuiltins

File: babel.config.js [hunk 2]
```diff
@@ -2,17 +2,12 @@ module.exports = {
...
     [
       '@babel/preset-env',
-      // @TODO: Do we need useBuiltIns?
-      // {
-      //   useBuiltIns: 'entry',
-      //   corejs: 2,
-      // },
+      {
+        useBuiltIns: 'entry',
+        corejs: 3,
```
Comment (scottalan) [inline babel.config.js]: and this..... is replacing it?

**Comment by scottalan** [comment]
Awesome! 👍

File: particle.js (hunk 1 above)
Comment (illepic) [reply particle.js]: Exactly! We no longer need to sneak in this entry point. Used to be super hacky, no longer hacky.

File: babel.config.js (hunk 2 above)
Comment (illepic) [reply babel.config.js]: This plus the new imports at the top of `apps/drupal/index.js` and `apps/node-pl/index.js`

**Pull request merged**